bundles/binding/org.openhab.binding.horizon/src/main/java/nl/kuijp/horizoncontrol/HorizonBox.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package nl.kuijp.horizoncontrol;
```

Comment:
Would it be possible to move this package to `org.openhab.binding.horizon.internal.control`?  Since it's source code directly in the binding, it would seem to make sense to have it live in the same package tree as the binding.